src/java/org/apache/cassandra/net/GlobalBufferPoolAllocator.java

```diff
@@ -36,6 +36,6 @@ private GlobalBufferPoolAllocator()
 
     static ByteBuf wrap(ByteBuffer buffer)
     {
-        return new Wrapped(instance, buffer);
+        return new Wrapped(instance, buffer, Integer.MAX_VALUE);
```

Comment:
nit: do you mind adding a comment saying this is the same as `AbstractByteBufAllocator#DEFAULT_MAX_CAPACITY`?